tests/sdk/v2/test_traversal.py

```diff
@@ -1136,3 +1136,54 @@ def test_metadata_on_dev(monkeypatch: pytest.MonkeyPatch):
     assert wf.metadata.sdk_version.minor == 42
     assert wf.metadata.sdk_version.patch == 0
     assert wf.metadata.sdk_version.is_prerelease
+
+
+def test_wf_default_imports():
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏Can you chop this test into multiple tests? Maybe using `@pytest.mark.parametrize`? In the current form, if one of the earlier scenarios fails then the later ones would not be tested.